src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -337,42 +139,39 @@ std::variant<Variable, ParsedQuery::Alias> SparqlQleverVisitor::visitTypesafe(
 }
 
 // ____________________________________________________________________________________
-ParsedQuery::Alias SparqlQleverVisitor::visitTypesafe(
-    SparqlAutomaticParser::AliasContext* ctx) {
+ParsedQuery::Alias Visitor::visitTypesafe(Parser::AliasContext* ctx) {
```

Comment:
All the `ParsedQuery::...` types can also be added to the using list above.
That way maybe it becomes visible, that they should maybe also live in the sparql_types namespace (but definitely not for this PR)